src/navigation/HomeContainer/HomeContainer.tsx

```diff
@@ -72,17 +79,40 @@ function HomeContainer(): React.JSX.Element {
   useLayoutEffect(() => {
     dispatch(hideSplashScreen(true)); // dismiss if not already dismissed when home screen shown.
   });
-
-  return (
-    <Stack.Navigator>
-      <Stack.Screen
-        name="Bookings"
-        component={BookingScreen}
-        options={{
-          headerRight: logoutButton,
-        }}
-      />
-    </Stack.Navigator>
-  );
+  console.log(featureFlags.tabBarEnabled);
+  if (featureFlags.tabBarEnabled === true) {
```

Comment:
Shouldn't need the === true. 

This should be:
```
if (featureFlags.tabBarEnabled) {
```

Reply:
Ahh does the work? I thought if you did that it just checked if it existed. Typescript isn't my first language so happy to remove the === true

Reply:
Sorry was accidentally logged in to my Sky account.

Reply:
Yes it should work. 

As you're probably aware if() is basically checking truthy values. In javascript this means...
```
0 // falsey
1 // truthy
(anything above 1) // truthy
null // falsey
undefined // falsey
```

From the above you can hopefully see what would pass in the if statement. 

Now with the `!` this inverts the value so `0` becomes `true` and the potential `undefined` (which could be returned due to your optional chaining is also `true`. 

Hope this makes sense?

Reply:
Thanks, makes sense and done!